### PR TITLE
MARBLE 1311  list creator 'unknown'  as 'Unknown Creator'

### PR DESCRIPTION
**Pull request by jmille79**

### CODE REVIEW

File: site/src/@ndlib/gatsby-theme-marble/components/Pages/MarbleItem/ItemLayout/TombstoneMetadata/index.js [hunk 1]
```diff
@@ -46,7 +46,8 @@ const TombstoneMetadata = ({ marbleItem }) => {
     },
   }
   const hasUri = typy(uriValue, '[0].value').safeArray.length > 0
-  const hasCreator = typy(creators, '[0].value').safeArray.length > 0
+  console.log(typy(creators, '[0].value').safeArray)
```
Comment (SteveMattison) [inline site/src/@ndlib/gatsby-theme-marble/components/Pages/MarbleItem/ItemLayout/TombstoneMetadata/index.js:49]: Do you want to include the console.log here?

**Review by SteveMattison — APPROVED** [review]

File: site/src/@ndlib/gatsby-theme-marble/components/Pages/MarbleItem/ItemLayout/TombstoneMetadata/index.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ const TombstoneMetadata = ({ marbleItem }) => {
     },
   }
   const hasUri = typy(uriValue, '[0].value').safeArray.length > 0
-  const hasCreator = typy(creators, '[0].value').safeArray.length > 0
+  const hasCreator = (typy(creators, '[0].value').safeArray.length > 0 && (typy(creators, '[0].value').safeArray[0] !== 'unknown'))
```
Comment (jonhartzler) [inline site/src/@ndlib/gatsby-theme-marble/components/Pages/MarbleItem/ItemLayout/TombstoneMetadata/index.js:49]: i don't think this is the correct algorithm we want. 
What we want is.  If there is a creator and it is exactly the string unknown.  Then we want it to be "Unknown Creator" 
The thing we don't want is the words unknown appearing like they are now when there is no creator and it does not make sense.  So we are moving the adding of the creator of unknow to the pipeline.

**Review by jonhartzler — CHANGES_REQUESTED** [review]

File: site/src/@ndlib/gatsby-theme-marble/components/Pages/MarbleItem/ItemLayout/TombstoneMetadata/test.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+import TombstoneMetadata from './'
+
+describe('TombstoneMetadata', () => {
+  test('Creator value is null', () => {
+    const item = {
+      id: 'ABC123',
+      description: 'How the Grinch Stole Christmas',
+      metadata: [
+        {
+          label: 'Creator',
```
Comment (SteveMattison) [inline site/src/@ndlib/gatsby-theme-marble/components/Pages/MarbleItem/ItemLayout/TombstoneMetadata/test.js:12]: Just so you know, I believe this scenario won't happen.  Either the Creator node will contain a display field with will either contain "unknown" or some valid value, or the Creator node will not exist in the metadata.

Comment (jmille79) [reply site/src/@ndlib/gatsby-theme-marble/components/Pages/MarbleItem/ItemLayout/TombstoneMetadata/test.js:12]: Thank you, that changes things a little bit. Fixing to reflect

**Review by jonhartzler — APPROVED** [review]

**Pull request merged**